dspace-sword/src/main/java/org/dspace/sword/SWORDUrlManager.java

```diff
@@ -530,4 +498,18 @@ public String getMediaLink(Bitstream bitstream)
             throw new DSpaceSWORDException(e);
         }
     }
+
+    /**
+     * Ensure configured paths when combined are vail URLs
+     *
+     * @param path the target SWORD endpoint
+     * @return a valid sword URL
+     */
+    private String validSWORDUrl(String path) {
```

Comment:
This method is confusing to me. It's not clear why we need to do all this string parsing.  It also seems like we can achieve this much easier by doing the following:
```
return dspaceUrl + "/" + swordPath + "/" + path;
```

It appears you may have done this just in case the `dspace.server.url` ends in a slash, but that should not occur...as we warn against that in our configuration docs & in dspace.cfg/local.cfg.